internal/config/cgmgr/cgroupfs.go

```diff
@@ -167,15 +160,24 @@ func setWorkloadSettings(cgPath string, resources *rspec.LinuxResources) error {
 		},
 	}
 
+	var paths map[string]string
 	if node.CgroupIsV2() {
-		var err error
-		mgr, err = fs2.NewManager(cg, cgPath, rootless.IsRootless())
-		if err != nil {
-			return err
-		}
+		paths := make(map[string]string, 1)
```

Comment:
this is a newly scoped variable, not the paths defined above

Reply:
oopsie )